public/app/routes/routes.tsx

```diff
@@ -37,6 +37,19 @@ export function getAppRoutes(): RouteDescriptor[] {
         () => import(/* webpackChunkName: "DashboardPage" */ '../features/dashboard/containers/DashboardPage')
       ),
     },
+    ...(config.featureToggles.publicDashboards
```

Comment:
This is mostly straight forward, but reading the conditional logic is a little ugly. At the bottom of this file there are function 

`  ...getPluginCatalogRoutes(),`

Just floating it that we could encapsulate that logic in one of those.. that being said, it's a single route. So maybe not worth it.

Another option could be to add it to the "extraRoutes" constant since that gets merged with the app routes

Reply:
Yeah that's fair. I just wanted it to be inline with all the other dashboard routes. I'll clean it up 👍